modules/get_sheduler.py

```diff
@@ -0,0 +1,8 @@
+from torch.optim import lr_scheduler
+
+def get(optimizer, sheduler_type, cycles):
```

Comment:
Just `get` is a very bad naming: get what? from where?

Also, you can use `**kwargs` here and pass them to corresponding scheduler for better control:
```suggestion
def some_meaningful_name(optimizer, sheduler_type, **scheduler_kwargs):
```